commit 968b7ca1feaa3acea68f43ee0b23aed18b00e48a
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-11T13:49:44-04:00 (3 months ago)
    
    TRITON-1807 papi makefile missing non-SmartOS dev npm block
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/Makefile b/Makefile
index 44370a7..9377361 100644
--- a/Makefile
+++ b/Makefile
@@ -45,6 +45,10 @@ NODE_PREBUILT_VERSION=v6.17.0
 ifeq ($(shell uname -s),SunOS)
         NODE_PREBUILT_TAG=zone64
         NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
+else
+	NPM=npm
+	NODE=node
+	NPM_EXEC=$(shell which npm)
 endif
 
 ENGBLD_USE_BUILDIMAGE	= true
